couchbase-on-ubuntu/couchbase-azure-install.sh

```diff
@@ -56,7 +56,7 @@ else
 fi
 
 #Script Parameters
-PACKAGE_NAME="couchbase-server-enterprise_3.0.3-ubuntu12.04_amd64.deb"
+PACKAGE_NAME="couchbase-server-enterprise_4.1.1-ubuntu12.04_amd64.deb"
```

Comment:
This one should probably be the same value as the parmater cbPackage to the template